src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -1162,13 +1167,36 @@ private function placeOrder(
         $this->sharedStorage->set('order', $order);
     }
 
+    private function getProductVariant(ProductInterface $product): ProductVariantInterface
+    {
+        /** @var ProductVariantInterface|null $variant */
+        $variant = $this->variantResolver->getVariant($product);
+
+        if ($variant === null) {
+            throw new \RuntimeException(sprintf('Product %s has no variant', $product->getName()));
```

Comment:
Just an idea, codes are easier to search.

```suggestion
            throw new \RuntimeException(sprintf('Product "%s" has no variant', $product->getCode()));
```